### ISSUE TITLE
Got an error if inspected object have no URL

### ISSUE THREAD

**Issue by maisim** [chienandalu/odooshow#1]
Using show on an object that has no url generates an error

```
ipdb> show(self.AccountAnalyticLine.search([]))
*** AttributeError: 'account.analytic.line' object has no attribute 'get_base_url'

```

**Comment by chienandalu** [comment]
Which version of Odoo are working on? From v13, this method is embeded in the models.Model abstract: https://github.com/odoo/odoo/blob/13.0/odoo/models.py#L3163

**Comment by maisim** [comment]
The error append on v12

**Comment by chienandalu** [comment]
Fixed in #2 

My support target is Python >=3.6 and Odoo >=13. In any case, if the fix is as trivial like this one, there's no problem in adding it :smile: :+1:
